### ISSUE TITLE
Variables with spaces do not render (`A B` is converted to A.B at data.frame step)

### ISSUE THREAD

**Issue by jeremiahpslewis** [hrbrmstr/taucharts#46]

**Comment by timelyportfolio** [comment]
Probably stepping on dangerous ground here, but could do something like this in [lines](https://github.com/hrbrmstr/taucharts/blob/master/R/taucharts.R#L28-L29).

``` r
oldnames <- names(data)
data <- data.frame(data)
names(data) <- oldnames
```

I would be concerned about downstream implications.

**Comment by hrbrmstr** [comment]
Fixed in `0.4.4.9001` (just now). Just had to do:

```
  data <- data.frame(data, check.names=FALSE)
```

instead of:

```
  data <- data.frame(data)
```
